minho/programmers/Level_1/7일차/신규_아이디_추천.js

```diff
@@ -0,0 +1,22 @@
+function solution(new_id) {
+    new_id = new_id
+        .toLowerCase() // 1단계
+        .replace(/[~!@#$%^&*()=+:?,<>/\[\]\{\}]/g, '') // 2단계
+        .replace(/\.{2,}/g, '.') // 3단계
+        .replace(/\.$|^\./g,''); // 4단계
+    console.log(new_id)
+    
+    if(!new_id){ // 5단계
+        new_id = 'a'; 
+    } 
```

Comment:
```suggestion
    .replace(/^$/,'a')
```
정규표현식으로도 빈 문자면은 a를 대입하는게 가능합니다!